pypaperless/models/common.py

```diff
@@ -14,6 +14,20 @@
 
 
 # custom_fields
+class CustomFieldExtraDataSelectOptions(TypedDict):
+    """Represent the `extra_data.select_options` field of a `CustomField`."""
+
+    id: str | None
+    label: str | None
+
+
+class CustomFieldExtraData(TypedDict):
+    """Represent the `extra_data` field of a `CustomField`."""
+
+    default_currency: str | None
+    select_options: list[CustomFieldExtraDataSelectOptions | None]
```

Comment:
IHMO it's `select_options: list[CustomFieldExtraDataSelectOptions] | None` and not not a list of 
`None`. ;-)